frontend/src/pages/landing-page/index.tsx

```diff
@@ -0,0 +1,542 @@
+                  determined by environmental and genetic factors using an
+                  interactive speed no-code UI. Understand published datasets or
+                  use them as a launchpad to identify new cell sub-types and
+                  states.
+                </p>
+                <div className={styles.linkContainer}>
+                  <Link href="/" passHref>
+                    <a>
+                      Explore a multi-tissue atlas
+                      <span className={styles.linkArrow}>
+                        <LinkArrow />
+                      </span>
+                    </a>
+                  </Link>
+                  <Link href="/" passHref>
+                    <a>
+                      See all the studies you can explore
+                      <span className={styles.linkArrow}>
+                        <LinkArrow />
+                      </span>
+                    </a>
+                  </Link>
+                </div>
+              </div>
+            </div>
+            <div className={styles.contentImageCol}>
+              <Image src={AnalyzeDatasetsImg} alt="" />
+            </div>
+          </div>
+
+          <div className={styles.contentRow} id="download-data" ref={section4}>
+            <div
+              className={`${styles.contentInfoCol} ${
+                inView4 ? styles.active : ""
+              }`}
+            >
+              <div className={styles.contentInfoFigureCol}>
+                <div className={styles.figureWrapper}>
+                  <DownloadDataIcon />
+                </div>
+                <span className={styles.figureSeparator}></span>
+              </div>
+              <div className={styles.contentInfoTextCol}>
+                <h2>Download and integrate data with zero wrangling</h2>
+                <p>
+                  Integrate datasets with zero data wrangling. Datasets with
+                  standard metadata annotations can be downloaded in AnnData and
+                  Seurat formats and custom cell selections from the complete
+                  corpus can be downloaded directly from R and python.
+                </p>
+                <div className={styles.linkContainer}>
+                  <Link href="/" passHref>
+                    <a>
+                      Browse datasets for download
+                      <span className={styles.linkArrow}>
+                        <LinkArrow />
+                      </span>
+                    </a>
+                  </Link>
+                  <span>
+                    Download with <a href="/">R</a> or <a href="/">Python</a>
+                    <span className={styles.linkArrow}>
+                      <LinkArrow />
+                    </span>
+                  </span>
+                </div>
+              </div>
+            </div>
+            <div className={styles.contentImageCol}>
+              <Image src={DownloadDataImg} alt="" />
```

Comment:
a11y: Can we add description of the image please 🙏 ? Thank you!

here and throughout